TESTS/integration/COMMON/common_defines_fs_test.h

```diff
@@ -50,15 +29,14 @@
 #elif COMPONENT_SD
 #define TEST_BLOCK_DEVICE_TYPE "SD"
 #define TEST_USE_FILESYSTEM FS_FAT
-#elif COMPONENT_FLASHIAP
```

Comment:
Hi @MarceloSalazar , as we discussed, if you can change the code like:
```
#elif COMPONENT_FLASHIAP && MBED_CONF_FLASHIAP_BLOCK_DEVICE_SIZE
#define TEST_BLOCK_DEVICE_TYPE "FLASHIAP"
```
And move it down the to last storage device that it try to use. that should meet both our intentions.
in the case user may want to use this device, they can define the FLASHIAP region in `mbed_lib.json` file